api/Document.json

```
@@ -4398,9 +4398,22 @@
           "support": {
             "chrome": [
               {
-                "version_added": "113",
-                "partial_implementation": true,
-                "notes": "Only available to Google Chrome's <a href='https://developer.chrome.com/docs/privacy-sandbox/first-party-sets/'>first-party sets</a>."
+                "version_added": "117"
```

Using the [mdn-bcd-collector](https://mdn-bcd-collector.gooborg.com/tests/api/Document/hasStorageAccess), I see that support was actually added by default in Chrome 119:

```suggestion
                "version_added": "119"
```

(Note: Chromestatus is often inaccurate, we have found.)

---

Imagine that's correct then. @johannhof can you confirm? And update chromestatus please?

---

This was a gradual rollout but it was first available in M117, so I think M117 is more correct as old versions should also have it, see https://chromestatus.com/feature/5085655327047680

---

I can't remember on MDN's policy on this - I could see arguments for both sides. I don't think it's worth adding a "Partial Support" from 117 and then updating  it to full support for 119 but could do if we want.

To me it's more important to note that it's well supported now so happy to go with either 117 or 119.

---

Once the rollout is done it's not partial anymore though. It's now fully rolled out to M117 and later, so any developer supporting M117+ will get the full SAA experience, not a partial support.

---

Ah in that case I can indeed see the argument for keeping this as 117. @queengooborg you agree?

---

When testing for support using Chrome versions available on BrowserStack, neither 117 nor 118 show support, so I feel that we should say 119 in BCD.  This is because a) the mdn-bcd-collector (which we use to update BCD entries) uses BrowserStack to obtain feature support information, and b) a contributor is likely to file an issue that the 117 is incorrect and 119 is because they tested in BrowserStack.

---

I wonder if BrowserStack freezes their 117 install to prevent Chrome auto updating, so it's not getting the latest experiment roll out updates? Or perhaps it's a snapshot and if left long enough it would update and get this functionality.

However, a real user on 117 would see this as working (providing they are not similarly frozen).

Anyway, I'm OK going with 119 as the time it was fully available. You OK with that @johannhof or feel strongly it should be set to 117 - in which case it may be best to add the partial support flag and a note, with full support arriving in 119.

---

Hmm yeah I can't get it to work on M117 either, maybe a side effect of how we rolled out. I'll follow up internally. Saying M119 is okay with me for now :)